packages/lib-classifier/src/plugins/tasks/models/Task.js

```diff
@@ -21,18 +21,20 @@ const Task = types.model('Task', {
     },
 
     get isComplete () {
-      return !self.required || !!self.annotation && self.annotation.isComplete
+      const annotation = tryReference(() => self.annotation)
+      return !self.required || !!annotation?.isComplete
     }
   }))
   .actions(self => ({
     updateAnnotation (value) {
-      self.annotation && self.annotation.update(value)
+      const annotation = tryReference(() => self.annotation)
+      annotation?.update(value)
     },
 
     complete () {
       // set an annotation for this task if there wasn't one already.
-      const { value } = self.annotation
-      self.updateAnnotation(value)
+      const annotation = tryReference(() => self.annotation)
+      self.updateAnnotation(annotation?.value)
```

Comment:
Do we actually want this action called if the annotation is undefined?

Reply:
I think this can be deleted completely. It doesn't create a new annotation, like the comment says.

Reply:
`updateAnnotation` was only being used to set up tests, so I've deleted it in favour of `annotation.update(value)` throughout the code.